### ISSUE TITLE
Screen Reader Improvements for Data Import Tool

### ISSUE THREAD

**Issue by ertay** [tprouvot/Salesforce-Inspector-reloaded#408]
**Is your feature request related to a problem? Please describe.**
There are a few improvements that can be done for screen reader users in the Data Import tool.
1. When the numbers on the 'Processing', 'Succeeded', 'Failed' checkboxes change, the screen reader user needs to check/uncheck the checkboxes in order to force a 'refresh' to read the current value. For example, I could import 250 records, and even though visually it would say 250 Succeeded, I need to click the Succeeded checkbox to for it to refresh for the screen reader so I can have it read the 250 Succeeded number. Otherwise it just says 0 succeeded.
2. When you click the button to start importing (e.g. Insert Account, Update contact etc.) a new dialog box shows up where you need to confirm that you are ready to begin your import. To a screen reader user, there is no indication that a new dialog has shown up. 
3. When the help dialog is enabled/disabled, there is no indicator to the screen reader that new information is now shown or hidden from the screen. Also screen reader focus does not move to the start of the help information when it is shown.

**Describe the solution you'd like**
The current values should be read to the screen reader without having the user to check/uncheck the checkboxes to force the value to refresh.
2. The expected behavior would be to move screen reader focus to the new dialog, read the message and maybe focus the button to confirm the start of the import.
There should be an alert to the screen reader that new information is shown on the screen and focus should move to this dialog.

**Describe alternatives you've considered**
Not applicable.

**Comment by tprouvot** [comment]
Hi @ertay,
Thanks for creating the issue, we'll do our best to improve the usability of the tool!

To help me on the testing, would you mind giving more info on the screen reader you're using ? Is it from Chrome / Firefox or your OS (MAC / Windows)?

**Comment by ertay** [comment]
Sure @tprouvot! The screen reader I am using is NVDA (version 2024.1). It's free and you can get it from https://www.nvaccess.org/download/?nvdaVersion=2024.1
I am using the Chrome extension of Inspector Reloaded.
Let me know if you have any other questions.
Thanks!

**Comment by tprouvot** [comment]
Hi @ertay 
Unfortunately I'm not able to install this product, is there any other tool compatible with mac that I could use ?

**Comment by ertay** [comment]
Hello @tprouvot, if you are on Mac, you can use VoiceOVer which is a screen reader that comes with the OS.

**Comment by tprouvot** [comment]
I'm very admirative for people who are using reloaded with screen reader, just tried few seconds and seems to be a real challenge !

**Comment by ertay** [comment]
Haha, that's probably because you've never used such a program before. Also, from what I have haerd from some VoiceOver users, VoiceOver has some issues with Salesforce pages in general. So something that works just fine with NVDA in Inspector may not work the same way with VoiceOver.
